src/View/Helper/CrudViewHelper.php

```diff
@@ -315,4 +329,28 @@ public function getViewVar($key = null)
     {
         return Hash::get($this->_View->viewVars, $key);
     }
+
+    /**
+     * Get css classes
+     *
+     * @return mixed
+     */
+    public function getViewClasses()
+    {
+        $action = $this->request->action;
+        $pluralVar = $this->getViewVar('pluralVar');
+        $viewClasses = (array)$this->getViewVar('viewClasses');
+        $args = func_get_args();
+
+        return implode(array_unique(array_merge(
```

Comment:
I think it's probably best for this method to return an array - and the view to implode it. Otherwise, any uses along the lines of "does this view have that class" would be a regex/strpos check.

Alternatively, rename the method to getCssClasses.